### PR TITLE
add scrollup@2.4.1 w/ git auto-update

### LINKED ISSUE #9934: [Request] Add jQuery ScrollUp
**Library name:**  jQuery ScrollUp
**Git repository url:** https://github.com/markgoodyear/scrollup
**npm package url(optional):** 
**License(s):**
**Official homepage:** http://markgoodyear.com/labs/scrollup/
**Wanna say something? Leave message here:** This library is used by many templates.

JSDelivr is hosting it here: http://www.jsdelivr.com/projects/jquery.scrollup

Thank you.

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by hare1039**
close #9934, cc @markgoodyear

@pvnr0082t Please review this PR. Thank you. I'm not sure about the file structure under `dist` so I choose only `.js` and `.min.js`.
Pull request for issue: #9934
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/markgoodyear/scrollup
 * Official website (optional, not the repository): https://markgoodyear.com/labs/scrollup/
 * NPM package url (optional):
 * License and its reference: MIT, ref: https://github.com/markgoodyear/scrollup/blob/master/package.json#L22
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 905
   - Count of watchers: 64
   - Count of forks: 285
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/scrollup/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/markgoodyear/scrollup.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "jquery.scrollUp.*"
```
Comment (PeterDaveHello) [inline ajax/libs/scrollup/package.json]: Let's use `"**/jquery.scrollUp.*"` under dist.

Comment (hare1039) [inline ajax/libs/scrollup/package.json]: Okey, I changed from `"jquery.scrollUp.*"` to `"**/jquery.scrollUp.*"`. Please review this PR again. Thanks.

**Comment by pvnr0082t** [comment]
Note that the file structure of versions before v2.4.0 are different from the latest version.

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Pull request merged**